src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -30,6 +31,6 @@ void npe2(String a, String b) {
 
  @Override
  public boolean equals(Object obj) {
-  return true;
+  return false;
```

Comment:
**Reporter**: PMD
**Rule**: OverrideBothEqualsAndHashcode
**Severity**: WARN
**File**: /home/bjerre/workspace/violations/violation-comments-to-github-plugin/work/workspace/pipe/src/main/java/se/bjurr/violations/lib/example/MyClass.java L34

Ensure you override both equals() and hashCode()&#10;&#10;&#10;Basic https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-465941356>*